packages/composites/apy-finance/test/integration/adapter.test.ts

```diff
@@ -39,7 +39,7 @@ describe('execute', () => {
       },
     }
 
-    it('should return success', async () => {
+    it.skip('should return success', async () => {
```

Comment:
Do we have a ticket to come back to this skipped test?

Reply:
I just created one: 16514